### ISSUE TITLE
Add location to education 

### ISSUE THREAD

**Issue by pastjean** [jsonresume/resume-schema#172]
Location is a nice option to add into education when there is multiple schools with same name in different cities!

```json
...
"education":[
  ...
  {
    ...
    "location": "quebec city"
    ...
  },
  ...
]
...
```

**Comment by jtwebman** [comment]
But they are the same school, right?

**Comment by pastjean** [comment]
forget the same name part , dunno why i wrote that. 

multiple school / degree each with its own city

**Comment by dbkaplun** [comment]
See also #63

**Comment by olivif** [comment]
should be fixed with #63, closing for now
